mongorestore/mongorestore.go

```diff
@@ -126,26 +127,6 @@ func New(opts Options) (*MongoRestore, error) {
 	return restore, nil
 }
 
-// SupportsCollectionUUID was removed from common/db/command.go, so copied to here
-func SupportsCollectionUUID(sp *db.SessionProvider) (bool, error) {
```

Comment:
FCV 3.6 isn't supported by mongorestore, and listCollections with atlas proxy would break this function. Best to remove it.

Reply:
I am waiting on confirmation from louisa that I can remove this.

Reply:
Seems that Ops Manager 5.0 does support FCV 3.4. However, --preserveUUID is only use for 3.6+. Since Ops Manager is the only user of this with 3.4, we should be able to remove this. Note, if anyone tries to run mongorestore with FCV 3.4 with --preserveUUID, the behavior will be undefined.